src/api/ledger/parse/pathfind.js

```diff
@@ -2,13 +2,23 @@
 'use strict';
 const parseAmount = require('./amount');
 
+// test fixture has an example response where source_amount is an IOU without
```

Comment:
Yeah, pretty sure. I looked at the code and its just basically
'amount.getJson(0)'. Ask again :)